File: LMS/LMS.Application/DTOs/HR/IncentiveDetailsDto.cs
using LMS.Domain.HR.Models;

namespace LMS.Application.DTOs.HR
{
    public class IncentiveDetailsDto
    {
        public Guid IncentiveId { get; set; }
        public Guid EmployeeId { get; set; }
        public string EmployeeName { get; set; }
        public decimal Amount { get; set; }
        public string Reason { get; set; } = string.Empty;
        public DateTime DecisionDate { get; set; }
        public bool IsPaid { get; set; }
        public bool IsApproved { get; set; }
        public string DecisionFileUrl { get; set; } = string.Empty;
    }
}

File: LMS/LMS.UI/huda-center/src/app/shared/models/hr/incentives/incentive-details.dto.ts
export interface IncentiveDetailsDto {
    incentiveId: string;
    employeeId: string;
    employeeName: string;
    amount: number;
    reason: string;
    decisionDate: string;
    decisionFileUrl: string;
    isApproved: boolean;
    isPaid: boolean;
}